zh_CN/rule/rule-engine.md

```diff
@@ -107,1255 +122,4 @@ SELECT * FROM "$events/client_connected"
 Dashboard 中提供了旧版 SQL 语法转换功能可以完成 SQL 升级迁移。
 :::
 
-## 规则引擎组成
```

Comment:
Where is this section?

Reply:
I removed this part. It is an overview of rule-engine, resource, action.